sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -777,8 +785,27 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 			if (!links[id].cpus->dai_name)
 				goto devm_err;
 		}
+		id++;
 	}
 
+	/* CNVi bt-offload */
+	if (sof_rt5682_quirk & SOF_BT_OFFLOAD_ENABLED) {
+		/* Codec is dummy */
+		links[id].id = id;
+		links[id].cpus = &cpus[id];
+		links[id].cpus->dai_name = "SSP2 Pin";
+		links[id].name = "SSP2-Codec";
```

Comment:
SSP2-NoCodec?

@ranj063 IIRC the dai link name is used in the topology, right? It may be useful to tag this link as different from the rest?

Reply:
should this be hardcoded to 2? Why not set it in the driver_data just like the rt5682/speaker SSP ports?

Reply:
yes @plbossart this dai_link name is used in topology but not specifically for SSP2 today at least for this machine. Why not call it SSP2-BT to be more specific instead of NoCodec?